misc/vault-integration/vault.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+if [[ -z "$VAULT_ADDR" || -z "$VAULT_NAMESPACE" || -z "$ROLE" ]]; then
```

Comment:
can we rename ROLE to VAULT_ROLE?

Reply:
[4905015](https://github.com/env0/templates/pull/120/commits/49050150d49fb447ded031f44b6e008bca74fed5)